lib/instance-validator.js

```diff
@@ -321,10 +321,7 @@ class InstanceValidator {
     let error;
 
     if (rawAttribute.allowNull === false && (value === null || value === undefined)) {
-      const validators = this.modelInstance.validators[field];
-      const errMsg = validators
-                      ? (validators.notNull || {}).msg
-                      : `${field} cannot be null`;
+      const errMsg = `${field} cannot be null`;
```

Comment:
I can fix it, but when you use 'notNull' validation, it fails. because of this (customization message)[https://github.com/sequelize/sequelize/blob/master/lib/utils/validator-extras.js#L81]

So end user can't use it anymore.